app/models/punk_api_client.rb

```diff
@@ -0,0 +1,35 @@
+class PunkAPIClient
+  PUNK_URL = "https://api.punkapi.com/v2/beers"
+
+  attr_reader :create_beers
+
+  def initialize(beers)
+    @create_beers = beers
+  end
+
+  def self.seed_recipes
+    return beer_request(PUNK_URL)
+  end
+
+  def self.beer_request(url)
+    response = Faraday.get(url)
+    parsed_response =  JSON.parse(response.body)
+    return create_beers(parsed_response)
```

Comment:
remove line 16 per the above

Reply:
have this method check the response to ensure its successful, then if it is return true or false accordingly